actors/verifreg/src/lib.rs

```diff
@@ -675,6 +734,7 @@ impl Actor {
         let st: State = rt.state()?;
         let mut claims = st.load_claims(rt.store())?;
         let mut updated_claims = Vec::<(ClaimID, Claim)>::new();
+        let mut prev_term_max_map: HashMap<ClaimID, ChainEpoch> = HashMap::new();
```

Comment:
Add a new member to the tuple value in `updated_claims` instead.

Reply:
We're not adding previous values anymore.